### PR TITLE
updated files for jenkins build

### PR DESCRIPTION
**Pull request by aq47**

### CODE REVIEW

**Comment by smillidge** [comment]
payara test please

**Comment by Pandrex247** [comment]
payara test please

**Comment by Pandrex247** [comment]
payara test please

**Comment by Pandrex247** [comment]
payara test please

**Comment by Pandrex247** [comment]
payara test please

**Comment by Pandrex247** [comment]
Do not merge, not successfully tested yet

**Comment by Pandrex247** [comment]
payara test please

File: appserver/tests/quicklook/adminconsole/src/test/admin/BaseAdminConsoleTest.java [hunk 1]
```diff
@@ -113,7 +113,7 @@ void setUpEnvironment(int port) {
     @Parameters({"admin.console.url", "amx.rmiport"})
     public void loginBeforeTest(String url, int port) throws Exception {
         this.adminUrl = url;
-        setUpEnvironment(port);
```
Comment (Pandrex247) [inline appserver/tests/quicklook/adminconsole/src/test/admin/BaseAdminConsoleTest.java]: Is it necessary to replace this variable with a hardcoded value?

File: appserver/tests/quicklook/adminconsole/src/test/admin/BaseAdminConsoleTest.java [hunk 2]
```diff
@@ -254,7 +254,7 @@ private MBeanServerConnection _getMBeanServerConnection()
         // CHANGE to RMI once it's working
         //
         // final String urlStr = "service:jmx:jmxmp://" + mHost + ":" + mPort;
-        final String urlStr = "service:jmx:rmi:///jndi/rmi://" + host + ":" + port + "/jmxrmi";
+        final String urlStr = "service:jmx:rmi:///jndi/rmi://" +  "127.0.0.1:5086"  + "/jmxrmi";
```
Comment (Pandrex247) [inline appserver/tests/quicklook/adminconsole/src/test/admin/BaseAdminConsoleTest.java]: Same as before, is it necessary to hard-code these?

**Comment by aq47** [comment]
Probably not as I was just fiddling around and it seemed to make the tests run but not fix the rmi connection refused problem. I mustve done other changes (hardcoded port values) as well. Sorry can't exactly pinpoint the fix.

**Comment by aq47** [comment]
I failed to see when and what the port value was being entered as arguement so resorted to hardcoding it. Bad practice - apologies for that

**Comment by aq47** [comment]
actually there was a weird solution on SO which made me do that - worked though

**Comment by smillidge** [comment]
Can this be merged yet?

**Comment by Pandrex247** [comment]
Not yet no, I haven't had a chance to make the required edits yet.

**Comment by Pandrex247** [comment]
Closing this pull request due to it not working on Jenkins.
See my updated pull request (435) which takes these changes and disables the failing tests.

**Pull request closed without merging**